src/TextInput/TextInput.svelte

```diff
@@ -120,7 +120,8 @@
           class:bx--visually-hidden="{hideLabel}"
           class:bx--label--disabled="{disabled}"
           class:bx--label--inline="{inline}"
-          class="{inline && !!size && `bx--label--inline--${size}`}"
```

Comment:
The `inline` condition isn't necessary since this element is already contained in an `inline` if block.